addons/mail/static/src/components/discuss/tests/discuss_inbox_tests.js

```diff
@@ -411,6 +411,79 @@ QUnit.test('"reply to" composer should send message if message replied to is not
     assert.verifySteps(['message_post']);
 });
 
+QUnit.test('show subject of mailing channel in inbox', async function (assert) {
+    assert.expect(3);
+
+    this.data['mail.message'].records = [{
+        author_id: [7, "Demo"],
+        body: "<p>Test</p>",
+        date: "2019-04-20 11:00:00",
+        id: 100,
+        is_discussion: false,
+        is_notification: false,
+        mass_mailing: true,
+        message_type: 'comment',
+        needaction: true,
+        needaction_partner_ids: [3],
```

Comment:
Reliance on magic number...
Please make session data explicit, by passing it to `env` from `start()`.